CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.0
```

Comment:
```suggestion
## 2.1.0-dev
```

I'd like to make sure we get this landed internally before publishing, and we've got another PR to merge and validate. Can we keep this as a `-dev` for now?

Reply:
done